zypp/KeyManager.cc

```diff
@@ -68,9 +68,27 @@ class KeyManagerCtx::Impl
   Impl();
   ~Impl();
 
-  std::list<std::string> verifyAndReadSignaturesFprs(const Pathname & file, const Pathname & signature, bool &verifed);
+  /** Return all fingerprints found in \a signature_r. */
+  std::list<std::string> readSignaturesFprs( const Pathname & signature_r )
+  { return readSignaturesFprsOptVerify( signature_r ); }
+
+  /** Tries to verify the \a file_r using \a signature_r. */
+  bool verifySignaturesFprs( const Pathname & file_r, const Pathname & signature_r )
+  {
+    bool verify;
```

Comment:
I know the called function does initialize it but it makes me nervous to see a uninitialized variable like that :D. Not a must but = false here will not hurt.

Reply:
done